orchestrator/api/voice_retell.py

```
@@ -45,6 +50,289 @@
+    # Gate 3 — the workspace's own toggle.
+    workspace = db.query(Workspace).filter(Workspace.id == ctx.workspace_id).first()
+    if workspace is None:
+        raise HTTPException(status_code=404, detail="Workspace not found")
+    ws_voice = live_settings.parse_workspace_voice_live(workspace.settings)
+    if not ws_voice.enabled:
+        logger.info("voice_live_mint_denied reason=workspace_disabled workspace=%s", ctx.workspace_id)
+        raise HTTPException(
+            status_code=403, detail="Auto Live is not enabled for this workspace"
+        )
+
+    # Gate 4 — the cap formula (ended + active×reserve ≥ cap refuses).
```

Ruff (0.15.21)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)